cephfs/select_fs.go

```diff
@@ -0,0 +1,33 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package cephfs
+
+/*
+#cgo LDFLAGS: -lcephfs
+#cgo CPPFLAGS: -D_FILE_OFFSET_BITS=64
```

Comment:
Where does that come from? Is that documented in the libcephfs docs?

Reply:
I think it was there before we started maintaining the project. I copy and pasted this part from the other cephfs go file(s). We can research if it's strictly needed but I'd keep it for now for consistencies sake.